src/main/kotlin/com/tally/kiosk/common/config/JpaConfig.kt

```
@@ -0,0 +1,11 @@
+package com.tally.kiosk.common.config
+
+import org.springframework.context.annotation.Configuration
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories
+
+@Configuration
+@EnableJpaAuditing
+@EnableJpaRepositories(basePackages = ["com.tally.kiosk.**.repository"])
+class JpaConfig {
+}
```

detekt (1.23.7)

[warning] The class or object JpaConfig is empty.

(detekt.empty-blocks.EmptyClassBlock)